internal/support/scheduler/router.go

```diff
@@ -0,0 +1,231 @@
+//
+// Copyright (c) 2018 Tencent
+//
+// Copyright (c) 2017 Dell Inc.
```

Comment:
Year is wrong

Reply:
updated the router header information to use the 2018 dell inc.